### DISCUSSION TITLE
Steady state model with density

### DISCUSSION THREAD

**Discussion by robinkeegan** [MODFLOW-ORG/modflow6#2058] [General]
I have a model that is a groundwater flow-transport model with density enabled in MODFLOW-6. I want to get a steady state head distribution with the salinity distribution held in its current position. I am holding the salinity distribution using the cnc package. Can I run the model in steady state? Or do you need to run it in transient for a windup period? It seems to run properly when I do it with storage disabled but I am not sure if this is valid with a coupled groundwater flow transport model with density. I know there is no steady state options for the groundwater transport model but I assume if it is held constant using the cnc package it is sort of in steady state.

**Comment by aprovost-usgs** [comment]
Hi Robin,

Running the simulation for a single time step with the flow model in steady state should give you the steady-state head distribution that corresponds to the initial concentration field.  If you use CNC to hold the concentrations at their initial values, the transport solution should reproduce the initial concentrations, but that shouldn't be necessary to get the flows you want, as they will already have been calculated.

Best,

-- Alden

Alden Provost
US Geological Survey
Integrated Modeling and Prediction Division
Model Support and Maintenance Branch



From: Robin Keegan-Treloar ***@***.***>
Sent: Friday, November 15, 2024 4:03 AM
To: MODFLOW-USGS/modflow6 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [MODFLOW-USGS/modflow6] Steady state model with density (Discussion #2058)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



I have a model that is a groundwater flow-transport model with density enabled in MODFLOW-6. I want to get a steady state head distribution with the salinity distribution held in its current position. I am holding the salinity distribution using the cnc package. Can I run the model in steady state? Or do you need to run it in transient for a windup period? It seems to run properly when I do it with storage disabled but I am not sure if this is valid with a coupled groundwater flow transport model with density. I know there is no steady state options for the groundwater transport model but I assume if it is held constant using the cnc package it is sort of in steady state.

-
